src/components/Modals/FiatRamps/types.ts

```diff
@@ -8,6 +8,8 @@ export type CreateUrlProps = {
   assetId: AssetId | string
   address: string
   fiatCurrency: string
+  fiatAmount?: string
+  amountCryptoPrecision?: string
```

Comment:
additive change to old fiat ramps, double-check whether or not that's an issue

Reply:
Confirmed "old" fiat ramps feature doesn't leverage those - conceptually not a fan of mixing and mashing old types if we can avoid this however @NeOMakinG

Reply:
This type is the args of the url creation util, which is supposed to support prefilling amounts, I don't see why we wouldn't update it to support amounts, how would you see that?

Reply:
Just by virtue of mixing old and new code, but mostly paranoia / clean separation of concerns standpoint, in effect at runtime it's safe so lezzgo!